tests/test_ncio.f90

```diff
@@ -16,19 +15,48 @@ program test_ncio
   integer ndim,nvar,ndims,ival,idate(6),ierr,n
   logical hasit
   
-  filename='data/dynf000.nc'
-  dset = open_dataset(filename)
-  
-  print *,'ncid=',dset%ncid
-  print *,'nvars=',dset%nvars
-  print *,'ndims=',dset%ndims
-  print *,'natts=',dset%natts
-  do ndim=1,dset%ndims
-     print *,'dim',ndim,trim(dset%dimensions(ndim)%name),dset%dimensions(ndim)%len,dset%dimensions(ndim)%isunlimited
-  enddo
-  do nvar=1,dset%nvars
-     print *,'var',nvar,trim(dset%variables(nvar)%name),dset%variables(nvar)%ndims,dset%variables(nvar)%dimlens
+  dsetin = open_dataset('dynf000_template.nc')
+  ! create a copy of the template file
+  dset = create_dataset('dynf000.nc',dsetin)
+  print *,'ncid=',dsetin%ncid
```

Comment:
What's with all the print statements?

Ideally tests will be terse. I usually write my tests to output something like:

*** Testing handling of some feature.
*** Testing ABC... ok
*** Testing XYZ... ok
SUCCESS!

The test should be testing all values internally, not printing them out for the user to see.

Reply:
fixed now